ports/stm32/mpbtstackport.c

```diff
@@ -64,14 +64,12 @@ STATIC const hci_transport_config_uart_t hci_transport_config_uart = {
 };
 
 void mp_bluetooth_hci_poll(void) {
-    if (mp_bluetooth_btstack_state == MP_BLUETOOTH_BTSTACK_STATE_OFF) {
+    if (mp_bluetooth_btstack_state == MP_BLUETOOTH_BTSTACK_STATE_OFF || mp_bluetooth_btstack_state == MP_BLUETOOTH_BTSTACK_STATE_HALTING) {
```

Comment:
with this change it seems that `btstack_run_loop_embedded_execute_once()` is no longer executed when in state `MP_BLUETOOTH_BTSTACK_STATE_HALTING`... does that matter?